src/components/Nav.jsx

```diff
@@ -14,23 +14,33 @@ const Nav = () => {
             alt="판다마켓 로고"
             className={styles["logo-img"]}
           />
-          <button className={styles.logo} type="button">
+          <button className={`btn ${styles.logo}`} type="button">
             판다마켓
           </button>
         </div>
-        <button className={`${styles.link}`} type="button">
+        <button className={`btn ${styles.link}`} type="button">
           자유게시판
         </button>
         <button
-          className={`${styles.link} ${
-            styles[`${location.pathname === "/items" ? "selected" : "none"}`]
+          className={`btn ${styles.link} ${
+            styles[
+              `${
+                location.pathname === "/items" ||
+                location.pathname === "/additem"
+                  ? "selected"
+                  : "none"
+              }`
+            ]
```

Comment:
조건문이 너무 복잡해보이네요! 리턴문은 최대한 명확하고 간결하게 유지해봅시다.

```jsx
  const isMarketPage =
    location.pathname === "/items" || location.pathname === "/additem";
```

```suggestion
          className={`btn ${styles.link} ${
            isMarketPage ? styles.selected : styles.none
          }`}
```

이정도로 정리해보는건 어떨까요? :)